lighthouse-core/gather/connections/cri.js

```diff
@@ -10,18 +10,20 @@ const WebSocket = require('ws');
 const http = require('http');
 const log = require('lighthouse-logger');
 
-const hostname = 'localhost';
+const DEFAULT_HOSTNAME = 'localhost';
 const CONNECT_TIMEOUT = 10000;
 const DEFAULT_PORT = 9222;
 
 class CriConnection extends Connection {
   /**
    * @param {number=} port Optional port number. Defaults to 9222;
+   * @param {string=} hostname Optional hostname. Defaults to localhost.
    * @constructor
    */
-  constructor(port) {
+  constructor(port, hostname) {
```

Comment:
I think this is an older file, so it doesn't use them yet, but do you want to switch these two to use default parameters instead of `||`?

Slight difference in `port` since it won't turn 0 into 9222, but we don't support port 0 at this stage anyways (trying to find a debugger on a random port makes no sense)